src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -1405,15 +1460,60 @@ bool AdaptDispatch::TabClear(const size_t clearType)
     switch (clearType)
     {
     case DispatchTypes::TabClearType::ClearCurrentColumn:
-        success = _pConApi->PrivateTabClear(false);
+        success = _ClearSingleTabStop();
         break;
     case DispatchTypes::TabClearType::ClearAllColumns:
-        success = _pConApi->PrivateTabClear(true);
+        success = _ClearAllTabStops();
         break;
     }
     return success;
 }
 
+bool AdaptDispatch::_ClearSingleTabStop()
```

Comment:
nit: these guys should have doc comments

Reply:
Right. I'll do that.

Reply:
Done in commit a0a4780bf0adff3a7829cd2467ce37a9adfd7860.